build/common-config.sh

```diff
@@ -85,7 +85,7 @@ var_set() {
 		env_val[PDO_DEFAULT_SIGCURVE]="${PDO_DEFAULT_SIGCURVE:-SECP384R1}"
 		env_desc[PDO_DEFAULT_SIGCURVE]="
 			PDO_DEFAULT_SIGCURVE is the ECDSA curve used by PDO for generating signatures.
-			Choose SECP384R1 for ccf. If not set, cyrpto library uses SECP256K1 as default which works for sawtooth"
+			Choose SECP384R1 for ccf. If not set, the crypto library uses SECP256K1 by default."
```

Comment:
perhaps remove "If not set, the crypto library uses SECP256K1 by default"?

Reply:
No, that's still the default.